Examples/Tests/electrostatic_sphere_eb/inputs_3d

```diff
@@ -33,5 +33,5 @@ eb_charge.type = ChargeOnEB
 eb_charge.intervals = 1
 eb_charge_one_eighth.type = ChargeOnEB
 eb_charge_one_eighth.intervals = 1
-// Select only one eighth of the sphere
```

Comment:
What will happen if someone uses this comment syntax now?

Reply:
It's a runtime error now. It used to be that `eb_charge_one_eighth.intervals` has multiple values, `1`, `//`, `Select`, etc.